docs/source/api/v4/cdns_id_notification.rst

```diff
@@ -0,0 +1,123 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-cdns-id-notification:
+
+****************************
+``cdns/{{ID}}/notification``
```

Comment:
Can you not GET the notification?

From [the API guidelines](https://traffic-control-cdn.readthedocs.io/en/latest/development/api_guidelines.html#get):

> _"All endpoints dealing with the manipulation or fetching representations of “Traffic Control Objects” MUST support this method."_

Reply:
Also, instead of ID did you consider using the Name instead? It's just much easier for humans to use

Reply:
yeah, you're right. name is probably a better option. i was just following the pattern but it looks like there is a pattern for both methods :)

Reply:
> Can you not GET the notification?

no, but i can add it

Reply:
> Can you not GET the notification?

a notification is attached to the cdn so you "get" it  by calling GET /cdns. think that's ok? i hate to add a route that has no value add.

Reply:
Well by the same token can you not just create a notification by updating the CDN with a PUT?